yt_idefix/fields.py

```diff
@@ -54,84 +47,63 @@ def setup_fluid_fields(self):
         )
 
 
-class PlutoXdmfFields(BaseFields):
-    _pres_units = "code_pressure"
-    _erg_units = "code_mass * (code_length/code_time)**2"
-    _rho_units = "code_mass / code_length**3"
-    _mom_units = "code_mass / code_length**2 / code_time"
-    _vel_units = "code_length/code_time"
-
+class PlutoFields(BaseFields):
     known_other_fields = (
+        # PLUTO 4 standard variable names normalized in upper case,
+        # Referring to Tools/IDL/pload.pro in Pluto v4.2-patch2
+        # Ion fractions are not listed here
         # Each entry here is of the form
         # ( "name", ("units", ["fields", "to", "alias"], # "display_name")),
-        ("rho", (_rho_units, ["density", "rho"], None)),
-        ("vx1", (_vel_units, ["vel_x"], None)),
-        ("vx2", (_vel_units, ["vel_y"], None)),
-        ("vx3", (_vel_units, ["vel_z"], None)),
-        ("prs", (_pres_units, ["prs", "pres", "pressure"], None)),
-        ("Bx1", ("code_magnetic", [], None)),
-        ("Bx2", ("code_magnetic", [], None)),
-        ("Bx3", ("code_magnetic", [], None)),
+        # alias should be set with yt-standard-name or pluto-macro-name
+        ("RHO", ("code_density", ["density"], None)),
+        ("VX1", ("code_velocity", ["velocity_x"], None)),
+        ("VX2", ("code_velocity", ["velocity_y"], None)),
+        ("VX3", ("code_velocity", ["velocity_z"], None)),
+        ("PRS", ("code_pressure", ["pressure"], None)),
+        ("AX1", ("code_magnetic", [], None)),
+        ("AX2", ("code_magnetic", [], None)),
+        ("AX3", ("code_magnetic", [], None)),
+        ("BX1", ("code_magnetic", [], None)),
+        ("BX2", ("code_magnetic", [], None)),
+        ("BX3", ("code_magnetic", [], None)),
+        ("EX1", ("code_magnetic/c", [], None)),
+        ("EX2", ("code_magnetic/c", [], None)),
+        ("EX3", ("code_magnetic/c", [], None)),
+        ("BX1S", ("code_magnetic", ["BX1s"], None)),
```

Comment:
Why the alias (`BX1S` to `BX1s`) ?

Reply:
`BX1s` is the macro defined in Pluto for variable `Bx1`, so are others (for staggered MHD). 

I set Pluto macro names as alias if they're different to upper case of the output names, so that users can call the fields with the names they use in Pluto.

Reply:
I found it makes unconsistency that some fields can be called with ("gas", ` pluto-macro-name`) while some fields can only be called with (`datatype`, `pluto-macro-name`). So I think `pluto-macro-name` should be set as alias for all of them, or let those macro names go instead. Which one do you prefer?

Reply:
ping @neutrinoceros again for a response (I guess my replies were missed :)

Reply:
indeed, sorry I often seem to miss stuff these days...
yes, these aliases, quite confusingly, are used to map to the `"gas"` field type, so it seems preferable to not have them at all.

Reply:
> so it seems preferable to not have them at all.

Ok I'll do it.